examples/cairo_threads/main.rs

```diff
@@ -117,18 +117,18 @@ fn build_ui(application: &gtk::Application) {
         }),
     );
 
-    ready_rx.attach(None, move |(thread_num, image)| {
-        let (ref images, ref origins, ref workers) = *workspace;
+    glib::MainContext::ref_thread_default().spawn_local(async move {
```

Comment:
This is GTK, you should use the global default main context for scheduling UI things

Reply:
I know, but the result is the same, and I just use this out of habit... will change it